newt/image/image.go

```diff
@@ -46,6 +46,10 @@ import (
 	"mynewt.apache.org/newt/util"
 )
 
+// Set this to enable RSA-PSS for RSA signatures, instead of PKCS#1
+// v1.5.  Eventually, this should be the default.
+var UseRsaPss = false
```

Comment:
I'm not quite sure how this would work. This is the point in the process where we are creating the image header, so none is available yet. Other arguments specify the version, for example.

Other than this option, the signing algorithm comes from the key that the user specified. However with v1.5 vs pss, the same keys are used, and we need some way to specify how this key should be used.